deb-package-builder/extensions/bitset/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ex
+source ${DEB_BUILDER_DIR}/functions.sh
+
+echo "Building bitset for gcp-php${SHORT_VERSION}"
+
+PNAME="gcp-php${SHORT_VERSION}-bitset"
+
+# Download the source
+if [ ${SHORT_VERSION} == "56" ]; then
+    download_from_pecl bitset 2.0.4
+else
+    download_from_pecl bitset 3.0.1
```

Comment:
How about to remove `3.0.1` here so that the builder will automatically follow the latest?

Reply:
Will do.